node/impl/full/eth_utils.go

```diff
@@ -31,6 +31,10 @@ import (
 	"github.com/filecoin-project/lotus/chain/vm"
 )
 
+const (
+	errcodeDefault = -32000
```

Comment:
What's this for ? Why did we have to add it ?

Reply:
Will remove it in favor of error code `3` for `EthCall` and `EstimateGas`, but in other places for execution reverted we need to return this(-32000) code.

Reply:
@akaladarshi But then why was it not a pre-existing code ? How did we handle this before you raised this PR ?

Reply:
I have to look into it little bit what exact error code FVM is returning, that's why I asked it in slack (do we have same error code as ethereum): https://filecoinproject.slack.com/archives/CP50PPW2X/p1728633105115299,

Will check this and update it accordingly.

Reply:
Irrespective of what it is returning, let's not change the error codes here. We only want the data field to be in like with ETH here -> everything else should stay the same right ?